requirements/min-reqs.old

```diff
@@ -33,20 +33,6 @@ scikit-image==0.19.0
 scipy==1.7.2
 zipfile-deflate64==0.2.0
 
-# docs
-ipywidgets==7.0.0
-nbsphinx==0.8.5
-sphinx==4.0.0
-
-# style
-black[jupyter]==21.8b0
-flake8==3.8.0
-isort[colors]==5.8.0
-pydocstyle[toml]==6.1.0
-pyupgrade==2.8.0
-
 # tests
-mypy==0.900
-nbmake==1.3.3
```

Comment:
We don't actually run any docs/style tests with the min version, no need to install those deps.